ui/src/main/java/org/pentaho/di/ui/spoon/Spoon.java

```diff
@@ -8945,36 +8921,6 @@ public Repository getRepository() {
   public void setRepository( Repository rep ) {
     this.rep = rep;
     this.repositoryName = rep != null ? rep.getName() : null;
-    try {
```

Comment:
This block remove some log messages and error dialog. How are we handling these cases in updated code

Reply:
This is just the same logic that already exists in MetastoreLocatorImpl.